rosdep/base.yaml

```
@@ -7425,6 +7425,11 @@ tesseract-ocr:
   fedora: [tesseract-devel]
   gentoo: [app-text/tesseract]
   ubuntu: [tesseract-ocr]
+tesseract-ocr-jpn:
+  arch: [tesseract-data-jpn]
+  debian: [tesseract-ocr-jpn]
+  fedora: [tesseract-langpack-jpn]
```

This is also available for RHEL: https://src.fedoraproject.org/rpms/tesseract#bodhi_updates

```suggestion
  fedora: [tesseract-langpack-jpn]
  rhel: [tesseract-langpack-jpn]
```

---

Thank you for your comment.
I added `rhel`.